### DISCUSSION TITLE
Tilde sign appears to high (between lines)

### DISCUSSION THREAD

**Discussion by Torinde** [dosbox-staging/dosbox-staging#4029] [General]
Staging 0.83.0-alpha (8d7c8)
![image](https://github.com/user-attachments/assets/17241acc-0f03-424e-b089-630a84c5b66a)

Virtualbox 
MS-DOS 6.22
![image](https://github.com/user-attachments/assets/98866df6-fe98-41a4-9798-82c4f2f57828)

PC DOS 2000
![image](https://github.com/user-attachments/assets/c09ff105-dd53-4cbe-8437-0f4ba2e9991a)

![image](https://github.com/user-attachments/assets/309aae45-6a0d-47f8-bc66-ae82bc5a1a2b)
![image](https://github.com/user-attachments/assets/819c6a47-6506-4e98-96cf-aa7633b000b6)

**Comment by FeralChild64** [comment]
Change the code page in _FreeDOS_, and you'll see the tilde higher too.

It's a _FreeDOS_ font author design decision - compare these two fonts (letter `S` vs letter `Ś` height) to see why it was done this way:

![MS-DOS font](https://github.com/user-attachments/assets/e1485617-2158-4e42-8a09-7e5861b20596)
![FreeDOS font](https://github.com/user-attachments/assets/c2cae27e-d9c6-41dd-9dec-25100e1b6d4a)

if you prefer the MS-DOS font, use the _MS-DOS_ CPI files to set the code page in _DOSBox Staging_.
